student-cvs/alifm-2843/main.mjs

```diff
@@ -0,0 +1,54 @@
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime.js";
+dayjs.extend(relativeTime);
+
+//* Toggle between adding and removing the "responsive" class to topnav when the user clicks on the icon */
```

Comment:
```suggestion
//* Toggle between adding and removing the "responsive" class to topnav when the user clicks on the icon */
```

Just as this comment is suggesting, to **Toggle** the class, there is a method precisely for that in the **classList** property:
https://developer.mozilla.org/en-US/docs/Web/API/Element/classList

Reply:
Hi!
Great! I'll take a look of the article you recommended, and try to improve my code according to it.
Thank you for letting me know about it!